node/hyperbahn/index.js

```diff
@@ -91,10 +92,19 @@ function HyperbahnClient(options) {
         'Must pass in top level tchannel');
     assert(options.tchannel.tracer, 'Top channel must have trace enabled');
     assert(options.serviceName, 'must pass in serviceName');
-    assert(Array.isArray(options.hostPortList),
-        'Must pass in hostPortList');
+    if (Array.isArray(options.hostPortList)) {
+        self.hostPortList = options.hostPortList;
+    } else if (typeof options.hostPortFile === 'string') {
+        var hostContent = safeSyncRead(options.hostPortFile);
+        assert(!hostContent.error, 'Failed to read the hostPortFile');
```

Comment:
Last discussion with @mranney ended with deciding that we don't in fact want to poll, watch, or in any way do anything fancy with the seed list file; let's discuss further offline if you have more to add to that conversation.